codec-http/src/main/java/io/netty/handler/codec/spdy/SpdyHeaders.java

```diff
@@ -150,11 +150,18 @@ private HttpNames() { }
     SpdyHeaders setObject(CharSequence name, Object... values);
 
     @Override
-    SpdyHeaders set(TextHeaders headers);
+    SpdyHeaders set(Headers<? extends CharSequence> headers);
 
     @Override
-    SpdyHeaders setAll(TextHeaders headers);
+    SpdyHeaders setAll(Headers<? extends CharSequence> headers);
 
     @Override
     SpdyHeaders clear();
+
+    /**
+     * {@link #get(CharSequence)} and convert the result to a {@link String}.
+     * @param name the name of the header to retrieve
+     * @return the first header value if the header is found. {@code null} if there's no such header.
+     */
+    String getAsString(CharSequence name);
```

Comment:
Is there a reason why `SpdyHeaders` and `StompHeaders` do not have some methods that `HttpHeaders` got? They look useful for SPDY and STOMP as well.

* `List<String> getAllAsString(CharSequence name);`
* `Iterator<Entry<String, String>> iteratorAsString();`
* `boolean contains(CharSequence name, CharSequence value, boolean ignoreCase);`

I wonder resurrecting `TextHeaders` would defeat the performance gain and simplicity or not. WDYT?